src/components/elements/input/MinutesDurationInput.tsx

```diff
@@ -9,6 +9,10 @@ type Props = { value?: number; onChange?: (val: number | null) => void } & Omit<
   'value' | 'onChange'
 >;
 
+export const minutesToHoursAndMinutes = (minutes: number) => {
+  return [Math.floor(minutes / 60), minutes % 60];
```

Comment:
maybe more accurate to use `as const` and avoid type widening? 
```suggestion
  return [Math.floor(minutes / 60), minutes % 60] as const;
```